unlock-app/src/pages/_error.js

```diff
@@ -1,34 +1,18 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import Router from 'next/router'
 
 import UnlockPropTypes from '../propTypes'
 import withConfig from '../utils/withConfig'
 
 class Error extends React.Component {
-  static getInitialProps({ res, err, req }) {
+  static getInitialProps({ res, err }) {
     const statusCode = res ? res.statusCode : err ? err.statusCode : null
 
-    // redirect to home now if not found
+    // why did we previously automatically redirect to home?
+    // at any rate, we cannot use Router here because Router doesn't exist server-side
```

Comment:
We should discuss this

Reply:
Maybe we can just adjust that function to be like the example here: https://stackoverflow.com/a/56928111

Reply:
I actually don't think it matters. We can just show an error if not found.

Reply:
(note that netlify's logic will do that _before_ it hits our app anyway)